src/components/application_manager/src/message.cc

```diff
@@ -32,6 +32,8 @@
 
 #include "application_manager/message.h"
 
+namespace smart_objects = NsSmartDeviceLink::NsSmartObjects;
```

Comment:
@wolfylambova22, this alias should present only in `smart_object.h` file.
As far as I understand, there still problems with this on `develop` branch, but after you switch on `feature/pre5_0_0_modification`, all should be fine. So please remove all changes which related to `smart_object` namespace.